### PR TITLE
Update toolchain vendor from unknown to nerves

### PR DESCRIPTION
**Pull request by mobileoverlord**
This also updates `nerves_toolchain_armv6_rpi_linux_gnueabi` to `nerves_toolchain_armv6_nerves_linux_gnueabi`

### CODE REVIEW

File: nerves_toolchain_ctng/build.sh [hunk 1]
```diff
@@ -156,7 +156,7 @@ elif [[ $BUILD_OS = "cygwin" || $BUILD_OS = "freebsd" ]]; then
     # For crosstool-ng
     export AWK=gawk
 else
-    echo "Unknown host OS: $BUILD_OS"
+    echo "nerves host OS: $BUILD_OS"
```
Comment (fhunleth) [inline nerves_toolchain_ctng/build.sh:159]:
```suggestion
    echo "Unknown host OS: $BUILD_OS"
```

File: nerves_toolchain_ctng/build.sh [hunk 2]
```diff
@@ -191,7 +191,7 @@ gcc_tuple()
     # Don't call this until after build_gcc()
     tuplepath=$(ls "$GCC_INSTALL_DIR")
     if [[ -e $tuplepath ]]; then
-        echo "unknown"
+        echo "nerves"
```
Comment (fhunleth) [inline nerves_toolchain_ctng/build.sh:194]:
```suggestion
        echo "unknown"
```

It feels like if somehow that call to `ls` fails that things should start failing.

File: nerves_toolchain_ctng/scripts/archive.sh [hunk 3]
```diff
@@ -27,7 +27,7 @@ gcc_tuple()
     # Don't call this until after build_gcc()
     tuplepath=$(ls "$GCC_INSTALL_DIR")
     if [[ -e $tuplepath ]]; then
-        echo "unknown"
+        echo "nerves"
```
Comment (fhunleth) [inline nerves_toolchain_ctng/scripts/archive.sh:30]:
```suggestion
        echo "unknown"
```

Same

**Comment by fhunleth** [comment]
What do you think of deleting all of those empty `.ex` files? They're kind of annoying to maintain.

**Comment by mobileoverlord** [comment]
I believe we can delete those files. If I remember correctly, they were a carry over to fix an old bug in Elixir that has long since been fixed.

**Review by fhunleth — APPROVED** [review]
This looks good to me. Before making a release, could we chat about updating the `_arm_` toolchain's name to `_armv7_` and the `armv5tejl` to `armv5`? That will align us with the Bootlin toolchain naming which I like for consistency. I'm 100% good with merging this unless CI finds something.

**Comment by mobileoverlord** [comment]
Sounds good on the rename. I'll merge this one and open a followup with those changes.

**Pull request merged**